pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/producer/PulsarKafkaProducer.java

```diff
@@ -77,6 +78,14 @@
     private List<ProducerInterceptor<K, V>> interceptors;
 
     private final Properties properties;
+    
+    public static class Config {
+        /**
+         * send operations will immediately fail with {@link ProducerQueueIsFullError} when there is no space left in
+         * pending queue.
+         **/
+        public static final String BLOCK_IF_PRODUCER_QUEUE_FULL = "block.if.producer.queue.full";
```

Comment:
Does this seem to be a special config setting that only works for pulsar Kafka producer? Can you please document this behavior?

Reply:
>> Does this seem to be a special config setting that only works for pulsar Kafka producer? 

Yes.

>> Can you please document this behavior?

Yes, added. can you please review it.